### ISSUE TITLE
WAN inference has garbled output when generating on a LORA with low rank (16)

### ISSUE THREAD

**Issue by Enyakk** [kohya-ss/musubi-tuner#139]
This is a weird one that took some time to hunt down. Whenever generating on a base LORA with rank 16 (alpha 16) the output is garbled. The amount of corruption can be different depending on the prompt but typically the first frame has major VAE-type errors which then decline during the video clip.

When I do the same on a LORA with rank 32 alpha 16 the output and samples are fine without corruption.

**Comment by Enyakk** [comment]
I just got a garbled sample output using
network_dim=32
network_alpha=16

--f 17

So, there is something more to this. I am trying to isolate the setting. Dropout possibly?

**Comment by Enyakk** [comment]
Ok, so I reinstalled the venv and this didn't help. I am at a loss.

This is the complete training command that produces the issue:
`accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 wan_train_network.py --task t2v-14B --dit "D:/ML/SD/models/Wan2.1/Comfy-Org-Wan_2.1_ComfyUI_repackages/diffusion_models/wan2.1_t2v_14B_fp8_e4m3fn.safetensors" --dataset "D:/ML/SD/workspace/musubi/2025-03-10/v4_rankdrop25/mixset_v2_aspect_512_wan.toml" --sdpa --mixed_precision bf16 --fp8_base --optimizer_type adamw8bit --learning_rate 2e-4 --gradient_checkpointing --max_data_loader_n_workers 2 --persistent_data_loader_workers --network_module networks.lora_wan --network_dim 32 --timestep_sampling shift --discrete_flow_shift 5.0 --max_train_epochs 16 --save_every_n_epochs 1 --seed 42 --output_dir D:/ML/SD/workspace/musubi/2025-03-10/v6 --output_name name-of-lora --sample_at_first --sample_prompts "D:/ML/SD/workspace/musubi/2025-03-10/v6/prompt.txt" --vae "D:/ML/SD/models/Wan2.1/Comfy-Org-Wan_2.1_ComfyUI_repackages/vae/wan_2.1_vae.safetensors" --t5 "D:/ML/SD/models/Wan2.1/Wan2.1-T2V-14B/models_t5_umt5-xxl-enc-bf16.pth"`
